load_confounds/strategies.py

```diff
@@ -238,28 +121,45 @@ class Params36Scrub(Confounds):
 
     """
 
-    def __init__(self, fd_thresh=0.2, std_dvars_thresh=3, demean=True):
+    def __init__(
+        self,
+        motion="full",
+        wm_csf="full",
+        scrub="full",
+        global_signal=False,
+        fd_thresh=0.2,
+        std_dvars_thresh=3,
+        demean=True,
+    ):
         """Default parameters."""
-        self.strategy = ["high_pass", "motion", "wm_csf", "scrub"]
-        self.motion = "full"
+        self.strategy = ["high_pass", "motion", "wm_csf", "scrub", "global"]
+        self.motion = motion
         self.n_motion = 0
-        self.wm_csf = "full"
-        self.scrub = "full"
+        self.wm_csf = wm_csf
+        self.scrub = scrub
+        self.global_signal = global_signal
         self.fd_thresh = (fd_thresh,)
         self.std_dvars_thresh = (std_dvars_thresh,)
         self.demean = demean
 
 
-class AnatCompCor(Confounds):
+class CompCor(Confounds):
     """
-    Load confounds using the aCOMPCOR strategy from Ciric et al. 2017.
-    Motion parameters (fully expanded), high pass filter, and acompcor.
+    Load confounds using the CompCor strategy from Behzadi et al., 2007.
+    Default with motion parameters (fully expanded), high pass filter, and anatomical compcor.
 
     Parameters
     ----------
     confounds_raw : Pandas Dataframe or path to tsv file(s), optionally as a list.
         Raw confounds from fmriprep
 
+    motion : string, optional
```

Comment:
it's really not ideal that we duplicate the docstrings this way, but I don't know how to do otherwise.

Reply:
Same here...